parser/testdata/comments_deps.go.mod

```diff
@@ -4,5 +4,5 @@ module example.com/user/project // end-line comment which should not break anyth
 go 1.13
 
 require (
-	github.com/mailru/easyjson v0.7.0
+	github.com/cyralinc/easyjson v0.7.10
```

Comment:
I wonder if we should take this approach, or if we should take the approach used in the benchmark/go.mod

The benefit of this approach is that we can make this completely independent, and we can expose these externally in other packages.

The downside is that it requires us to update this every time we update the version.

Reply:
@billprovince I think this is okay. `comments_deps.go.mod` is a test go.mod used by easyjson's parser test (https://github.com/cyralinc/easyjson/blob/pramodc/use-cyralinc/parser/pkgpath_test.go#L19). Basically, easyjson binary, reads a go.mod as part of its functionality. This parser_test tests that functionality using these test go.mods